deeppavlov/models/torch_bert/torch_transformers_squad.py

```diff
@@ -184,7 +184,9 @@ def __call__(self, features: List[InputFeatures]) -> Tuple[List[int], List[int],
         b_input_ids = torch.cat(input_ids, dim=0).to(self.device)
         b_input_masks = torch.cat(input_masks, dim=0).to(self.device)
         b_input_type_ids = torch.cat(input_type_ids, dim=0).to(self.device)
-        
+        if self.pretrained_bert == 'roberta-base':
```

Comment:
it seems like we should generalize this condition (not only to roberta-base)

Reply:
updated